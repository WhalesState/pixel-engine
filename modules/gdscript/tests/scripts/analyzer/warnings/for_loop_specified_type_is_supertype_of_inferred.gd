func test():
	var a: Array[Node2D] = []
	for node: Node in a:
		print(node)
