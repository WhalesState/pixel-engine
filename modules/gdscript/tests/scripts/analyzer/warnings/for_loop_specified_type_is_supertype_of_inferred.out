GDTEST_OK
>> WARNING
>> Line: 3
>> REDUNDANT_FOR_VARIABLE_TYPE
>> The for loop iterator "node" has inferred type "Node2D" but its supertype "Node" is specified.
